### PR TITLE
Options, Payment, Firewall, CLI

### PR DESCRIPTION
**Pull request by Bani2791**
1. Setup> Settings > Options

https://bani-2024-01-5--connexcs-docs.netlify.app/setup/settings/options/#__tabbed_1_3

2. Payment 

https://bani-2024-01-5--connexcs-docs.netlify.app/customer/payment/

Added (Description): Please note add a # symbol for starting a private comment to store private information along with the public information.

3. Firewall 

https://bani-2024-01-5--connexcs-docs.netlify.app/setup/advanced/firewall/

Added: You can only enter a single IP

4. CLI Modifiers

https://bani-2024-01-5--connexcs-docs.netlify.app/customer/cli/#use-pre-asserted-identity-as-cli

### CODE REVIEW

File: docs/customer/cli.md [hunk 1]
```diff
@@ -359,9 +359,12 @@ In the Rewrite CLI section, `44` is the literal 44 digits, and `$1` will contain
 
 You can use all or part of a sent Pre-Asserted-Identity as the CLI. The following example shows how to capture them all.
 
-| CLI            | Pre-Asserted-ID  | Rewrite CLI | Rewrite P-Asserted-ID | Forced | Use DID  | Userspace DB |
+| CLI| Pre-Asserted-ID  | Rewrite CLI | Rewrite P-Asserted-ID | Forced | Use DID  | Userspace DB |
 |----------------|------------------|-------------|-----------------------|--------|----------|--------------|
-| `.*`           | `^(?<paid>.*)$`  | `$<paid>`   |                       | No     | Disabled | None         |
+| `.*`| `^(?<paid>.*)$`| `$<paid>`|| No| Disabled | None|
+| `$0`| `^(?<paid>.*)$`| `$<paid>`|| No| Disabled/Filter | Select a database|
+| `/0`| `^(?<paid>.*)$`| `$<paid>`|| No| Disabled/Filter | Select a database|
```
Comment (AdamNawaz-connex) [inline docs/customer/cli.md:366]: not correct

File: docs/customer/cli.md [hunk 2]
```diff
@@ -359,9 +359,12 @@ In the Rewrite CLI section, `44` is the literal 44 digits, and `$1` will contain
 
 You can use all or part of a sent Pre-Asserted-Identity as the CLI. The following example shows how to capture them all.
 
-| CLI            | Pre-Asserted-ID  | Rewrite CLI | Rewrite P-Asserted-ID | Forced | Use DID  | Userspace DB |
+| CLI| Pre-Asserted-ID  | Rewrite CLI | Rewrite P-Asserted-ID | Forced | Use DID  | Userspace DB |
 |----------------|------------------|-------------|-----------------------|--------|----------|--------------|
-| `.*`           | `^(?<paid>.*)$`  | `$<paid>`   |                       | No     | Disabled | None         |
+| `.*`| `^(?<paid>.*)$`| `$<paid>`|| No| Disabled | None|
+| `$0`| `^(?<paid>.*)$`| `$<paid>`|| No| Disabled/Filter | Select a database|
```
Comment (AdamNawaz-connex) [inline docs/customer/cli.md:365]: not correct

File: docs/customer/cli.md [hunk 3]
```diff
@@ -359,9 +359,12 @@ In the Rewrite CLI section, `44` is the literal 44 digits, and `$1` will contain
...
+| `/0`| `^(?<paid>.*)$`| `$<paid>`|| No| Disabled/Filter | Select a database|
+| `db`| `^(?<paid>.*)$`| `$<paid>`|| Yes|| Select a database|
```
Comment (AdamNawaz-connex) [inline docs/customer/cli.md:367]: not correct

File: docs/customer/cli.md [hunk 4]
```diff
@@ -394,13 +397,13 @@ This will BLOCK all USA Freephone numbers
...
 |-------------|:--------------:|:-----------:|:-------------------------------------------------|
-| 123456789   |          ^1234 |             |   Allows only numbers starting with 1234 to pass |
-| 123456789   |           789$ |             |      Allows only numbers ending with 789 to pass |
-| 12345678912 | ^[0-9]{11,12}$ |             | Allows only numbers with 11 or 12 digits to pass |
-| +123456789  |   ^\+([0-9]+)$ |          \1 |                                 Remove the leading + |
-| 01782123456 |    ^0([1-9]+)$ |        44\1 |             Remove the leading 0 and replace it with 44 |
+| 123456789 | ^1234 ||Allows only numbers starting with 1234 to pass |
+| 123456789|789$ ||Allows only numbers ending with 789 to pass |
+| 12345678912 | ^[0-9]{11,12}$ || Allows only numbers with 11 or 12 digits to pass |
+| +123456789  |^\+([0-9]+)$ |\1 |Remove the leading + |
```
Comment (AdamNawaz-connex) [inline docs/customer/cli.md:405]: Its showing as ^+([0-9]+)$

File: docs/setup/settings/options.md [hunk 5]
```diff
@@ -38,3 +38,7 @@ Configure **Options** to set key default behaviors for the system. Click **`Edit
     Click on the `Save` button to apply the changes to your account.
 
     <img src="/setup/img/siprewrite.png">
+
+=== "Custom Pages"
+
+   Custom Pages allow you to add the created Apps to the **Dashboard** and the **Anyedge**.
```
Comment (AdamNawaz-connex) [inline docs/setup/settings/options.md:44]: They are at the bottom on general in options and provide some info both separately dashboard and  AnyEdge Stats

File: docs/customer/payment.md [hunk 6]
```diff
@@ -15,6 +15,8 @@ To add a new payment
     + **Description** is the name of the payment.
         If you wish to hide any information on the customer portal, use **#** followed by the text to hide.
         For example, PayPal 87878787744XXX #payment from the demo.
```
Comment (AdamNawaz-connex) [inline docs/customer/payment.md:17]: Keep in new paragraph and add to example  only 'PayPal 87878787744XXX' will be visible on the customer portal, as the '#payment from the demo' part is hidden.

File: docs/customer/cli.md (hunk 4 above)
Comment (Bani2791) [reply docs/customer/cli.md:405]: corrected

File: docs/setup/settings/options.md (hunk 5 above)
Comment (Bani2791) [reply docs/setup/settings/options.md]: done
 Custom Pages allow you to add the created Pages from the IDE to the **Dashboard** and the **Anyedge**. It provides respective information on the Dashboard and Anyedege.

File: docs/customer/payment.md (hunk 6 above)
Comment (Bani2791) [reply docs/customer/payment.md:18]: !!! Example
            PayPal 87878787744XXX #payment from the demo.
        !!! Note
            Add a **#** symbol for starting a private comment to store private information along with the public information.

File: docs/customer/cli.md (hunk 2 above)
Comment (Bani2791) [reply docs/customer/cli.md]: done

**Pull request merged**